src/BlazorWebFormsComponents/BuildTool.tt

```diff
@@ -0,0 +1,45 @@
+﻿<#@ template debug="false" hostspecific="true" language="C#" #>
+<#@ output extension=".cs" #>
+<#@ assembly name="System.IO"#>
+<#@ import namespace="System.IO" #>
+<#@ import namespace="System.Text" #>
+<#@ import namespace="System.Collections" #>
+<#@ import namespace="System.Collections.Generic" #>
+
+namespace BlazorWebFormsComponents
+{		
+	public static class Constants
+	{
+<#
+
+const string FunctionPattern = " function(";
+
+			string text;
+			var fileName = @"E:\Dev\OpenSource\BlazorWebFormsComponents\src\BlazorWebFormsComponents\wwwroot\js\Basepage.js";
```

Comment:
Let's make this a relative path to the JavaScript we're going to analyze

Reply:
Done. Worked out how to get the working folder where the .tt file is located.